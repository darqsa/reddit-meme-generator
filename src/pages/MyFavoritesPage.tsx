import Header from "../components/Header/Header";

const MyFavoritesPage = (): JSX.Element => {
  return (
    <>
      <Header currentPage={"favoritesPage"} />
    </>
  );
};
export default MyFavoritesPage;
