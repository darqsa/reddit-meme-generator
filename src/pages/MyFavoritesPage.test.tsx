import { BrowserRouter } from "react-router-dom";
import TestRenderer from "react-test-renderer";
import MyFavoritesPage from "./MyFavoritesPage";
describe("Given a MyFavoritesPage page", () => {
  describe("When its rendered on screen", () => {
    test("Then it should match the snapshoot", () => {
      const expectedMyFavoritesPage = TestRenderer.create(
        <BrowserRouter>
          <MyFavoritesPage />
        </BrowserRouter>
      );

      expect(expectedMyFavoritesPage).toMatchSnapshot();
    });
  });
});
