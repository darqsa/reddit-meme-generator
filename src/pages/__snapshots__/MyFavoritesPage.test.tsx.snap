// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a MyFavoritesPage page When its rendered on screen Then it should match the snapshoot 1`] = `
Array [
  <div
    className="header-container"
  >
    <header
      className="sc-hKMtZM cklVyW header-container__title-container"
    >
      <h1
        className="sc-jSMfEi kZvA-DQ header-container__title"
      >
        FAVORITE 
        <span
          className="sc-papXJ eRKGXE"
        >
          MEMES
        </span>
      </h1>
    </header>
    <section
      className="sc-gKXOVf iqGkus header-container__section-container"
    >
      <div
        className="sc-iBkjds eITUJN header-container__texts-container"
      >
        <p
          className="sc-ftvSup jccDpC header-container__text"
        >
          Here you can find your selected favorite memes! Click delete to remove a meme from your list.
        </p>
      </div>
      <div
        className="sc-jqUVSM jOaidU header-container__buttons-container"
      >
        <a
          href="/home"
          onClick={[Function]}
        >
          <button
            className="sc-dkzDqf wbqHG"
            onClick={[Function]}
          >
            Main Page
          </button>
        </a>
      </div>
    </section>
  </div>,
  <div
    className="sc-gsnTZi EaBay"
  >
    <ul
      className="meme-list"
    />
  </div>,
]
`;
